src/libpriv/rpmostree-core.c

```diff
@@ -2244,10 +2244,6 @@ apply_rpmfi_overrides (int            tmp_metadata_dfd,
           g_str_equal (group, "root"))
         continue;
 
-      if (!S_ISREG (mode) &&
-          !S_ISDIR (mode))
-        continue;
```

Comment:
Hmm, can we leave this and instead augment it with `S_ISLNK`? E.g. it's technically possible for the RPM to contain block devices, right? (And plus the unpacker uses `ignore_unsupported_content`).